mobile/status.go

```diff
@@ -737,12 +795,8 @@ func sendTransactionWithChainID(chainID int, txArgsJSON, password string) string
 	return prepareJSONResponseWithCode(hash.String(), err, code)
 }
 
+// SendTransaction converts RPC args and calls backend.SendTransaction.
 func SendTransaction(txArgsJSON, password string) string {
-	return logAndCallString(sendTransaction, txArgsJSON, password)
```

Comment:
Let's not nightmare the git history there and back 😄 
Keep the private func and the public wrapper for overall consistency in the file.
```suggestion
    return sendTransaction(txArgsJSON, password)
```

Same for other cases in this file.

Reply:
fixed😅